ExampleGallery/Effects/ExampleGallery.Effects.csproj

```diff
@@ -93,17 +93,18 @@
     <Prefer32Bit>true</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
-    <!-- A reference to the entire .Net Framework and Windows SDK are automatically included -->
-  </ItemGroup>
-  <ItemGroup>
     <Compile Include="ExampleVideoEffect.cs" />
     <Compile Include="DisplacementEffect.cs" />
     <Compile Include="RotatedTilesEffect.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="VariableVideoEffect.cs" />
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This causes a build break due to multiple different framework versions being referenced simultaneously  (and the change is not needed)